knexfile.js

```diff
@@ -0,0 +1,29 @@
+module.exports = {
+
+  development: {
+    client: 'sqlite3',
+    connection: {
+      filename: './data/lambdaNotes.sqlite3'
+    },
+    useNullAsDefault: true,
+    migrations: {
+      directory: './data/migrations'
+    },
+    seeds: {
+      directory: './data/seeds'
+    }
+  },
+  testing: {
+    client: 'sqlite3',
+    connection: {
+      filename: './data/testNotes.db3',
+    },
+    useNullAsDefault: true,
+    migrations: {
+      directory: './data/migrations',
+    },
+    seeds: {
+      directory: './data/seeds',
+    },
+  },
+};
```

Comment:
All of your node modules have bloated the review page, making it extremely laggy. Add these to your gitignore next time.